pkg/custompluginmonitor/types/config.go

```diff
@@ -141,5 +141,22 @@ func (cpc CustomPluginConfig) Validate() error {
 		}
 	}
 
+	for _, rule := range cpc.Rules {
```

Comment:
maybe a `validateDefaultConditionExists` method ?

Reply:
I think either way is fine. I will just keep it here as I feel this is still simple.